app/src/main/java/org/sopt/cgv/feature/seats/component/SeatsScreenTopBar.kt

```diff
@@ -0,0 +1,154 @@
+package org.sopt.cgv.feature.seats.component
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+import org.sopt.cgv.R
+import org.sopt.cgv.core.designsystem.theme.CGVTheme
+import org.sopt.cgv.core.designsystem.theme.Gray850
+import org.sopt.cgv.core.designsystem.theme.White
+
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun SeatsScreenTopBar(
+    modifier: Modifier = Modifier,
+    viewModel: SeatSelectViewModel,
```

Comment:
```suggestion
    selectedTimeCardIndex: Int,         
    onTimeCardClick: (Int) -> Unit,     
    onBackClick: () -> Unit             
```
여기서도 ViewModel 의존성을 제거하고 필요한 상태와 이벤트 콜백만 매개변수로 받도록 변경하면 좋을 거 같아요! 이유는 위와 같습니당